src/views/accompagnement/liste.vue

```diff
@@ -30,6 +30,8 @@
       >Se connecter</a
     >
     <div v-if="accompagnements">
+      <SimulationSearch />
```

Comment:
```suggestion
      <SimulationSearch v-if="loggedIn" />
```

Sinon il sera affiché dans tous les cas ; la variable `accompagnements` est toujours set après un call API au sondage

Reply:
Je viens de push le correctif :)